lambda-src/lambda_function.py

```diff
@@ -0,0 +1,23 @@
+import json
+import urllib.parse
+import boto3
+
+print('Loading function')
+
+s3 = boto3.client('s3')
+
+def lambda_handler(event, context):
+    # print("Received event: " + json.dumps(event, indent=2))
```

Comment:
remove this comment code

Reply:
it was only for testing and maybe I need it later for more tests. How would you do it?